### PR TITLE
Bootstrap basic login using user-management service

### LINKED ISSUE #17: Bootstrap Basic UI using React

### LINKED ISSUE #24: Bootstap-UI using react js

### PR DESCRIPTION
**Pull request by rasmitha-koduru**

### CODE REVIEW

File: WeatherApp/public/index.html [hunk 1]
```diff
@@ -19,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>WeatherApp</title>
+    <title>React App</title>
```
Comment (adithyaselv) [inline WeatherApp/public/index.html]: Why React App ?

File: WeatherApp/src/Home.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Home() {
+  return (
+    <div>
+      Welcome to the Home Page!
```
Comment (adithyaselv) [inline WeatherApp/src/Home.js]: Welcome to CAPtivate

**Review by adithyaselv — DISMISSED** [review]
very minor comments

File: WeatherApp/public/index.html (hunk 1 above)
Comment (rasmitha-koduru) [reply WeatherApp/public/index.html]: Thanks, forgot to change while merging

**Review by adithyaselv — APPROVED** [review]

**Pull request merged**